00-RELEASENOTES

```diff
@@ -1,3 +1,53 @@
+Redis 6.2 RC3   Released Tue Feb 1 14:00:00 IST 2021
+================================================================================
+
+Upgrade urgency LOW: This is the third Release Candidate of Redis 6.2.
+
+Here is a comprehensive list of changes in this release compared to 6.2 RC2,
+each one includes the PR number that added it, so you can get more details
+at https://github.com/redis/redis/pull/<number>
```

Comment:
```suggestion
at https://github.com/redis/redis/pull/8427
```

Reply:
that text suggests you look up the specific issue (mentioned at the end of each bullet), not this PR (this wasn't the intent).
if we want to refer the user to a version with hyperlinks we can refer him to https://github.com/redis/redis/releases

Reply:
Ah, never mind